docs/manage-persistent-storage.md

```diff
@@ -0,0 +1,391 @@
+# Manage persistent storage
+Like many other databases, etcd stores its state on disk. In a default deployment (as described in [deploy-etcd](./deploy-etcd.md)),
+the filesystem attached to charmed etcd will be removed when charmed etcd is removed. The content of the etcd database
+would then be lost.
+
+To decouple the lifecycle of the etcd database from the lifecycle of its deployment, charmed etcd provides a feature 
+called persistent storage. It allows to keep storage volumes around, even after a deployed etcd application or unit 
+has been removed.
+
+The use cases can be broken down into two groups:
+- reusing storage from previous units but within the same etcd cluster/database (see: [Same cluster scenario](#same-cluster-scenario))
+or
+- reusing storage from another etcd cluster/database (see: [Different cluster scenario](#different-cluster-scenario))
+
+Charmed etcd uses two different storage volumes:
+- `data` containing the raw data files (the actual database) written and managed by etcd
+- `logs` for logfiles written by etcd
+
+The following document will explain how to use this feature in charmed etcd.
```

Comment:
I wouldn't qualify this as a feature, rather "how to reuse storage volumes" or "storage reuse support" (applies to the rest of the doc)
As this may set certain expectations, most of which are juju related